src/components/10-atoms/icon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.1.0
```

Comment:
If there are breaking changes in here, this version upgrade needs to be a major one.